### PR TITLE
[DC-1786] [DC-1788] RT fitbit validation notebook updates

### PR DESCRIPTION
**Pull request by jqian2015**

### CODE REVIEW

**Review by nishanthpp93 — COMMENTED** [review]
Overall works as expected but can be a bit slow, so any minor optimization can help with the time. Let me know your thoughts.

File: data_steward/analytics/rt_cdr_qc/cdr_deid_qa_report6_fitdata.py [hunk 1]
```diff
@@ -22,207 +22,435 @@
...
 FROM `{project_id}.{pipeline}.pid_rid_mapping` m
-JOIN `{project_id}.{non_deid}.activity_summary` i
+JOIN `{project_id}.{non_deid_fitbit}.steps_intraday` i
 ON m.person_id = i.person_id
 ),
 
 df2 AS (
 SELECT d.person_id,
-CONCAT(d.person_id, '_', DATE_ADD(d.date, INTERVAL m.shift DAY)) as d_newc
+CONCAT(d.person_id, '_', DATE_ADD(d.datetime, INTERVAL m.shift DAY)) AS d_newc
```
[the remark this answers is not in the document]
Comment (jqian2015) [reply data_steward/analytics/rt_cdr_qc/cdr_deid_qa_report6_fitdata.py:272]: i updated and thanks for the advice. as for datetime_add, i tested but did not see the difference, so did not change. let me know your thoughts.

[the remark this answers is not in the document]
Comment (nishanthpp93) [reply data_steward/analytics/rt_cdr_qc/cdr_deid_qa_report6_fitdata.py:272]: Thanks, you're right it works.

**Review by nishanthpp93 — APPROVED** [review]
Tested it against the RT fitbit and it ran without issues.

**Pull request merged**